res/openvpn-gui-res-it.rc

```diff
@@ -77,7 +77,7 @@ END
 ID_DLG_CHALLENGE_RESPONSE DIALOG 6, 18, 212, 72
 STYLE WS_SIZEBOX | WS_POPUP | WS_VISIBLE | WS_CAPTION | DS_CENTER | DS_SETFOREGROUND
 EXSTYLE WS_EX_TOPMOST
-CAPTION "OpenVPN - Challenge Response"
+CAPTION "OpenVPN - Risposta challenge"
```

Comment:
I think it was previously agreed to not translate this text, because this is the actual name of the functionality